.github/workflows/build.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         with:
           args: release --snapshot --skip-publish --rm-dist
       - name: Scan image for vulnerabilities
-        uses: docker://docker.io/aquasec/trivy:0.2.1
+        uses: docker://docker.io/aquasec/trivy:0.4.0
```

Comment:
I wonder if this could just use trivy:latest

Reply:
Yup. In this context using the latest tag is perfectly valid.